tasks/main.yml

```diff
@@ -46,6 +47,26 @@
   when: cluster_venv != None
 
 - block:
+    - name: Extract server ids from cluster group output
+      set_fact:
+        cluster_servers: "{{ cluster_group.output_value | json_query('[*].nodes[*].id') | flatten }}"
+        cluster_interfaces: "{{ cluster_net | map(attribute='net') | list }}"
+
+    - name: Attach interfaces to servers
+      os_server_interface:
```

Comment:
I think this needs to run before 'Reset the python interpreter', to ensure we have access to the packages in the virtualenv.

Reply:
Done